### ISSUE TITLE
UI for a comparison of Submissions accross tasks / datasets

### ISSUE THREAD

**Issue by mam10eks** [tira-io/tira#477]
I saw a very nice way to visualize submissions across multiple tasks / datasets in the entity linking tool [GERBIL](https://aksw.org/Projects/GERBIL.html).

The UI of Gerbil had the following main features for this:
- Select datasets (default: all)
- Select submissions (default: all)
- Select measures (forgot what the default was, maybe all?)

For this, Gerbil rendered a table with the main results but also some figures (e.g., task/dataset on the x-axis sorted by decreasing effectiveness and effectiveness on the y-axis resulting in a line plot.)

**Comment by mam10eks** [comment]
This would be especially interesting for TIRA/TIREx (cc. @potthast, @heinrichreimer, @seanmacavaney).

@juhehehe, @Kavlahkaff, If you want, you can already start a bit with brainstorming on the UI for this.

**Comment by potthast** [comment]
Can you post some screenshots? It's not clear to me where to find them on the linked page.

**Comment by mam10eks** [comment]
I also could not find the screenshots anymore.

Here is a sketch with what I remember from the talk:
![20230908_123526](https://github.com/tira-io/tira/assets/10050886/6bc823ed-f334-4160-ba18-4e945f7036cf)


Does this sketch clarify the idea?

I would really love to have something like this.
Maybe this would be a good next todo for @juhehehe?

**Comment by mam10eks** [comment]
So in the end it is basically a table.
There was also a way to draw the content of the table in a figure, but the table would be the main thing to me.

**Comment by heinrichreimer** [comment]
Yes, I agree, comparing the results of a bunch of selected approaches in this table-like fashion would also be the main use case for me.

Further nice-to-have's:
- CSV export
- online significance tests (for starters just a t-test with fixed alpha, that automatically applies multiple test correction based on the number of selected approaches)

**Comment by mam10eks** [comment]
I finished the REST endpoint for this and integrated it into the existing RunList.vue:

![Screenshot_20230911_124313](https://github.com/tira-io/tira/assets/10050886/83ef18dd-6d3d-4f4b-999c-ea8d3a250dbd)

From that, I think @juhehehe can finalize the UI of this.
(You see this table on the submission pages)
